pages/latest.jsx

```diff
@@ -1,10 +1,6 @@
 import React from 'react';
-import ProjectList from '../components/project-list/project-list.jsx';
+import ProjectLoader from '../components/project-loader/project-loader.jsx';
 
-export default React.createClass({
-  render() {
-    return (
-      <ProjectList />
-    );
-  }
-});
+export default function () {
```

Comment:
same future proof question. Also, clearly latest.jsx is identical to featured.jsx except for the featured flag, so I'm not sure there's much point in creating separate components for them - why don't we just use `<ProjectLoader/>` and `<ProjectLoader featured={true}/>` where we now import these?